reference/employer/openapi.yaml

```diff
@@ -1018,12 +1018,50 @@ paths:
                       description: The display name of the payroll provider.
                     products:
                       type: array
-                      description: The list of Finch products supported on this payroll provider.
+                      description: '[DEPRECATED] The list of Finch products supported on this payroll provider. This field is now deprecated. Please use `authentication_methods` instead.'
                       items:
                         type: string
                     category:
                       type: string
                       description: 'The categroy of the integration. Possible values: `hris`, `ats`'
+                    authentication_methods:
+                      type: array
+                      x-stoplight:
+                        id: kqh1orpm8c2uh
+                      description: The supported authentication methods for this provider.
+                      items:
+                        x-stoplight:
+                          id: wa4yuci2z09fa
+                        type: object
+                        properties:
+                          id:
+                            type: string
+                            x-stoplight:
+                              id: z1v5i0z0fygnl
+                            format: uuid
+                          type:
+                            type: string
+                            x-stoplight:
+                              id: zo0rd28yaha2t
+                            enum:
+                              - credentials
+                              - oauth
```

Comment:
currently in our code base oauth + api_token = API. we won't be able to know which one is api_token and which one is oauth just from db

Reply:
okay, we should figure out how to differentiate between the two before we launch this.

Reply:
wait, we never tried to differentiate them by design when we started it. why can't they both be API in this case since 1 is to provide API token directly, one is use oauth to get API token automatically?

Reply:
yeah i don't really care how its implemented internally. it may be that from the api-sever perspective, both just use an api token, but from the user's perspective they are quite different methods. this is an external interface so the developer knows which providers require the customer to manually follow instructions and generate and input an api token vs going through oauth. there's a big difference in user experience between the two that the developer may want to know about.